cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -151,15 +166,31 @@ func (i *deployCommand) InitHook() {
 	i.command.Flags().StringVar(&i.flags.imageVersion, "image-version", "", "Image version for standard Kogito build images. Ignored if a custom image is set for image-s2i or image-runtime.")
 	i.command.Flags().StringVar(&i.flags.mavenMirrorURL, "maven-mirror-url", "", "Internal Maven Mirror to be used during source-to-image builds to considerably increase build speed, e.g: https://my.internal.nexus/content/group/public")
 	i.command.Flags().BoolVar(&i.flags.enableIstio, "enable-istio", false, "Enable Istio integration by annotating the Kogito service pods with the right value for Istio controller to inject sidecars on it. Defaults to false")
+	i.command.Flags().BoolVarP(&i.flags.fromFile, "from-file", "f", false, "If set, a build from a single file, no pom.xml needed, will start, you can provide a entire directory, a single file from filesystem or url, the supported files are dmn, drl, bpmn or bpmn2.")
```

Comment:
Since this is a `bool`, I can write the command like:

```
kogito deploy test myprocess.bpmn --env ENV1=VALUE  --from-file --maven-mirror-url="https:/"
```
Do you think that it makes sense from a UX perspective? 

Why not a `string`?